generate.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+JAVA=$(which java)
+
+if [[ -x "$JAVA" ]]
+then
+    CONTAINER_ID=$(docker create datadog/docker-library:openapi-generator-cli-60b29e1f8e2)
```

Comment:
can we use the image name defined in `docker-compose.yaml`?

Reply:
Not sure what you mean by that. You mean read the image name from the compose file?